product_variant_cost_price/models/product_product.py

```diff
@@ -55,6 +55,12 @@ def _set_standard_price(self, product, value):
 
     @api.model
     def create(self, values):
+        if 'product_tmpl_id' in values and 'standard_price' not in values:
+            template = self.env['product.template'].browse(
+                values.get('product_tmpl_id'))
+            values.update({
```

Comment:
OK, I see, so it's not a problem of the history, but the variant cost price. Thanks for clarifying.